src/app/public/modules/repeater/fixtures/repeater.component.fixture.ts

```diff
@@ -41,6 +41,10 @@ export class RepeaterTestComponent {
 
   public showRepeaterWithActiveIndex = false;
 
+  public showRepeaterWithNgFor = false;
+
+  public items: string[] = [ 'Item 1', 'Item 2', 'Item 3' ];
```

Comment:
Sorry, just a code style nitpick - would you mind putting `items` up a bit the properties are in alphabetical order?

Reply:
No problem, moved it up.